modules/aruco/CMakeLists.txt

```diff
@@ -1,2 +1,3 @@
 set(the_description "ArUco Marker Detection")
 ocv_define_module(aruco opencv_core opencv_imgproc opencv_calib3d WRAP python java)
+ocv_warnings_disable(CMAKE_CXX_FLAGS -Wunused-parameter)
```

Comment:
Disabling warnings globally should be avoided.
- use `CV_UNUSED(unused_parameter)` macro
- or avoid parameter name in declaration "..., int /*unused_paramer*/, ... " (doesn't work in case of conditional compilation with "#if").

Reply:
oh, sorry... my bad, i was trying to get rid of 4244. It is ok if i disable 4244 globally?

Reply:
btw, thank you for your help :)

Reply:
4244 is "conversion from 'type1' to 'type2', possible loss of data"

OpenCV has static code scans and disabling of these warnings doesn't help really (static code analyzers has own settings).
So it is better to extend type of target variable (if it make sense and doesn't affect performance), or make an explicit conversion (like double -> float).